tools/crdb-physical-replication.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+
+# Intended to automate https://docs.google.com/document/d/1vTX6N72J0yL2dDVdyJg063gC761QdZcMQ3ADthFYi8g/edit
+
+bold=$(tput bold)
+normal=$(tput sgr0)
+
+echo "${bold}CockroachDB Physical Replication setup${normal}"
+echo ""
+echo "This script will assist you in running a demo of Physical Replication on your local machine"
+echo ""
+
+echo "Checking prerequisites..."
+versionstr=$(cockroach version 2>&1)
+
+if [[ $versionstr == *"command not found"* ]]; then
+    echo "❌ We did not find CockroachDB installed"
+    echo "Please begin by installing CockroachDB: https://www.cockroachlabs.com/docs/stable/install-cockroachdb.html"
+    echo "Re-run this script after you have done so"
+    exit 2
+fi
+
+regexmajor=(v[0-9]+\.[0-9]+)
```

Comment:
This is waaaay overengineered (and also specific on a version of bash the customer may not be running)

Try this:
```sh
if ! version_str=$(cockroach version --build-tag); then ... fi

version_str=${version_str#v}
major=${version_str%%.*}
# Uncomment the following if we ever need to check the minor version. For now, we don't.
# version_str=${version_str#${major}.}
# minor=${version_str%%.*}

if [ $major -lt 23 ]; then
   echo "you will need to install ..."
fi
```

Reply:
Done